src/NetworkManager.cpp

```diff
@@ -72,6 +74,23 @@ NetworkManager::NetworkManager() : httpClient(true), downloadClient(true)
 		LUA->Release(L);
 	}
 
+	RageFile f;
+	if(f.Open(SpecialFiles::CA_BUNDLE_PATH))
+	{
+		RString data;
+		f.Read(data);
+		f.Close();
+
+		this->tlsOptions.caFile = data;
+	}
+	else
+	{
+		LOG->Warn("Reading '%s' failed: %s", SpecialFiles::CA_BUNDLE_PATH.c_str(), f.GetError().c_str());
+	}
+
+	this->httpClient.setTLSOptions(this->tlsOptions);
```

Comment:
Nothing to change now, but in general I don't like using `this->` to reference class members/functions, but since the members aren't particularly identifiable it's okay for now. Normally I suffix members with `_` (which is also the Google style guide) to help differentiate them from local vars..